app/models/Harvest.scala

```diff
@@ -47,6 +47,17 @@ case class Harvest(id: Int,             // DB key
       .on('updated -> newDate, 'id -> id).executeUpdate()
     }
   }
+
+  def harvestUrl = {
```

Comment:
Feels less so to me, but doesn't matter. I might suggest that the method be called 'oaiHarvestUrl' or some such, since its protocol-specific. I had a vague idea that harvest was an abstraction, and the protocol specifics would be elsewhere. But realistically OAI-PMH is all we are looking at for the foreseeable future